socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -93,6 +94,7 @@ panic_abort::
 PR_WaitCondVar
 RaiseException
 RealMsgWaitFor
+__rust_alloc_error_handler
```

Comment:
This has two underscores at the beginning, but I see both `_rust_alloc_error_handler` and `__rust_alloc_error_handler` in crash reports, so I'd add both versions.

Example with single underscore: bp-e17483cd-347e-46fb-ab61-fdc320210131

Example with double underscore: 11f6b902-efa9-4861-9781-70d4d0210201

Reply:
Oh, you did this by editing the file in GitHub. I'll land this and then add the second part.